includes/discovery/sensors/state/asa.inc.php

```diff
@@ -31,20 +39,37 @@
         ];
         create_state_index($state_name, $states);
 
+        $stateLookupTable = [
```

Comment:
There is no reason to have a lookup table.  Just change the snmp query to give numeric values.

Best to convert it to SnmpQuery if you can too.

Reply:
Fixed both!